### ISSUE TITLE
NTR: Dihydrothymine in urine high (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8815]
New term request Dihydrothymine in urine high (segal_060511203412):   UMLS: 1847705

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000119
- label: Elevated urinary dihydrothymine level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: 
- def: The amount of dihydrothymine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8815
- synonyms: Dihydrothymine in urine high(NONE)
